docs/modules/ROOT/pages/index.adoc

```diff
@@ -852,179 +850,116 @@ The `validFrom` field MUST exist and it MUST express the date and time when the
-
-.Example:
-[source,json]
-----
-"credentialSubject": {
-  ...
-  "affiliations": [
+      },
+      "verifiedAt": "2024-07-26T22:30:15Z"
+    },
     {
-      "id": "https://example-affiliated-organization.com",
-      "name": "Example Affiliated Organization",
-      "proof": "https://example-affiliated-organization.com/proofs/ck4592p5lk8u05mdg8bg5ac7ishlqfh1"
-    }
-  ]
-  ...
-}
-----
-
-[#vc-credentialsubject-affiliation-id]
-Affiliation ID:::
-The `affiliations[?].id` MUST be present and MUST be a valid URI that, when dereferenced, MUST result in a proof of authenticity of the _<<_named_actor,named actor's>>_ affiliated entity.
-
-[#issue-149]
-NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/149[issue #149]): Define what we mean by "proof of authenticity" here.
-
-[#vc-credentialsubject-affiliation-name]
-Affiliation name:::
-The `affiliations[?].name` MUST be present and MUST NOT be empty. The `affiliations[?].name` property is the name of the  _<<_named_actor,named actor's>>_ affiliated entity.
-
-[#vc-credentialsubject-affiliation-proof]
-Affiliation proof:::
-The `affiliations[?].proof` is an OPTIONAL field. If present, it MUST be a valid URI that, when dereferenced, MUST result in the proof of authenticity of the  _<<_named_actor,named actor's>>_ affiliation validity, i.e., the _<<_named_actor,named actor>> is affiliated indeed with this organization. The content as well as the process of validatig an affiliation relation are outside the scope of this specification.
-
-[#vc-credentialsubject-identities]
-===== Identities
-
-The `identities` property is an optional field within the `credentialSchema` property. If present, it MUST be an array containing information about the accounts that the _<<_named_actor,named actor>>_ requested to be bound with the _<<C2PA asset>>_ being described.
-
-.Example:
-[source,json]
-----
-"credentialSubject": {
-  ...
-  "identities": [
+      "type": "cawg.affiliation",
+      "provider": {
+        "id": "https://example-affiliated-organization.com",
+        "name": "Example Affiliated Organization",
+        "proof": "https://example-affiliated-organization.com/proofs/ck4592p5lk8u05mdg8bg5ac7ishlqfh1"
+      },
+      "verifiedAt": "2024-07-26T22:29:57Z"
+    },
     {
-      "type": "socialProvider",
+      "type": "cawg.social_media",
+      "name": "Silly Cats 929",
       "username": "username",
       "uri": "https://example-social-network.com/username",
       "provider": {
         "id": "https://example-social-network.com",
         "name": "Example Social Network"
       },
-      "createdAt": "2024-05-27T08:40:39.569856Z"      
+      "verifiedAt": "2024-05-27T08:40:39.569856Z"
+    },
+    {
+      "type": "cawg.crypto_wallet",
+      "username": "username",
```

Comment:
I think we should use address property for crypto wallets and remove the uri property

Reply:
See earlier discussion on this question. Could be persuaded if there's a strong reason for it.